gradle.properties

```diff
@@ -182,15 +182,15 @@ pom.organization.url=http://chrisdoyle.com/
 pom.issueManagement.system=GitHub
 pom.issueManagement.url=https://github.com/chrisdoyle/gradle-fury/issues
 
-pom.ciManagement.system=
-pom.ciManagement.url=
+#pom.ciManagement.system=Travis
+#pom.ciManagement.url=https://travis-ci.org/chrisdoyle/gradle-fury
 
 pom.scm.url=https://github.com/chrisdoyle/gradle-fury
 pom.scm.connection=https://github.com/chrisdoyle/gradle-fury.git
 pom.scm.developerConnection=https://github.com/chrisdoyle/gradle-fury.git
 
-pom.distributionManagement.site.id=
-pom.distributionManagement.site.url=
+#pom.distributionManagement.site.id=github
+#pom.distributionManagement.site.url=https://github.com/chrisdoyle/gradle-fury/releases
```

Comment:
ahh i added it for testing, then commented it out for testing. most of those elements are not allowed to have empty strings or 0 children